editor/loadFile.js

```diff
@@ -0,0 +1,26 @@
+function createFileLoader (moonchild, channel) {
+  var fileLoader = Object.create(null);
+
+  // assuming that the editor doesn't get changed
+  // weird stuff will happen if the editor -does- get changed
+  var editor = moonchild.getEditor()._codeMirror;
```

Comment:
I think it would be better to just implement a `setValue` method on the editor (or maybe call it `setFileContents` or something).